### ISSUE TITLE
How do you send a mail with for example 'Hello World' to your subscriber?

### ISSUE THREAD

**Issue by bian22** [spatie/laravel-newsletter#210]

**Comment by userabuser** [comment]
You would want to use the `createCampaign` method:

```php
Newsletter::createCampaign(
  /*
    string $fromName,
    string $replyTo,
    string $subject,
    string $html = '',
    string $listName = '',
    array $options = [],
    array $contentOptions = []
*/
);
````

Then `getApi` method to call underlying MailChimp API provided by [`drewm/mailchimp-api`](https://github.com/drewm/mailchimp-api) (I use this regularly myself, it's good 👍)

```php
$api = Newsletter::getApi();

// https://usX.api.mailchimp.com/3.0/campaigns/3e06f4ec92/actions/send
$campaign_id = 3e06f4ec92;
$send = $api->post("campaigns/$campaign_id/actions/send");

```

This is a very basic example just to put you on the right track. If you want to send one-off/unique campaign a workflow might look like this:

1. get list of subscribers you want to mail
2. if they are from several lists, create a temporary list for this campaign and add them
3. create the campaign
4. send the campaign to the temporary list you have created

This can effectively emulate transnational email if you have unique/bespoke one off emails.

Or if this is a regular thing, e.g. create campaign and send to an already segment list that you build through some other process then the same steps apply but rather than creating a new list purely for grouping users for a one time email, you would use the existing list of your choice.

Basically, drewm/mailchimp-api library exposes under the hood all of the following: 

https://developer.mailchimp.com/documentation/mailchimp/reference/overview/

So anything not covered by this laravel package, can be access easily just one method call away.

Hope that helps.
